README.md

```diff
@@ -56,6 +56,49 @@ module "spacelift_workerpool" {
 }
 ```
 
+### FedRAMP Example
+
+If your account is in the Spacelift FedRAMP environment, set the `is_fedramp` variable to `true`:
```

Comment:
@adamconnelly On second thought, do we need the whole example or just this one line. We might want to have a full working example in the `examples` folder instead of cluttering the README file.